src/initialization/initializeJavaScript/cleanups.ts

```diff
@@ -1,4 +1,6 @@
-import { prompt } from "enquirer";
+import enquirer from "enquirer";
+
+const prompt = enquirer.prompt;
```

Comment:
[Question] Ehh, I'm not thrilled about needing to go through the default import here. Why would it be necessary? Or: there's got to be a way to set up the tooling to allow it, no?

Reply:
_sigh_

```plaintext
file:///Users/josh/repos/TypeStat/lib/initialization/sources/index.js:1
import { prompt } from "enquirer";
         ^^^^^^
SyntaxError: Named export 'prompt' not found. The requested module 'enquirer' is a CommonJS module, which may not support all module.exports as named exports.
CommonJS modules can always be imported via the default export, for example using:

import pkg from 'enquirer';
const { prompt } = pkg;
```